openlane_update_tool/action.yml

```diff
@@ -0,0 +1,45 @@
+name: Update OpenLane tool.
+
+description: Update a tool used in OpenLane flow to the latest commit hash.
+
+inputs:
+  tool_name:
+    description: Name of the tool to update.
+    required: true
+  pdk_root:
+    description: Path to the PDK folder.
+    default: pdks
+  build_ol:
+    description: >
+      Build docker image after tool update. Only disable this option if you
+      intend to update multiple tools, this way you only need to build the
+      final OpenLane image with all the tools once.
+    default: true
+  ol_tag:
+    description: Tag to use when building the docker image.
+    default: current
+
+runs:
+  using: composite
+  steps:
+
+    - name: Update tool hash
+      shell: bash
+      run: python3 .github/scripts/update_tools.py ${{ inputs.tool_name }}
+
+    - name: Build docker image for updated tool
+      shell: bash
+      run: |
+        make -C docker PDK_ROOT=${{ inputs.pdk_path }} \
```

Comment:
do you know if openlane images are built with `BUILDKIT_INLINE_CACHE=1`, you should be able to benefit from the cache using https://docs.docker.com/engine/reference/commandline/build/#specifying-external-cache-sources if that's the case

Reply:
They do not, at least `grep BUILDKIT_INLINE_CACHE` returns nothing.